packages/components/calendar/src/calendar-base.tsx

```diff
@@ -98,6 +101,7 @@ export function CalendarBase(props: CalendarBaseProps) {
         key={`calendar-month-${i}`}
         currentMonth={currentMonth.month}
         direction={direction}
+        renderCellContent={renderCellContent}
```

Comment:
you can directly retrieve `renderCellContent` from context in `calendar-cell.tsx`. therefore, there is no need to pass it all the way from base.